browser/vendor/THREE.MeshLine.js

```diff
@@ -0,0 +1,430 @@
+THREE.MeshLine = function() {
```

Comment:
This is an extra copy of THREE.MeshLine.js. Also the real copy should really be under /vendor/spite not vendor/three